<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoverKTS</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="displayTabWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>671</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="plotTab">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savePlotButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>500</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadPlotButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>530</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="phiCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Phi</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="wyCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wy</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ayCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ay</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="wzCB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>520</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wz</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="azCB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>540</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Az</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="psiCB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>500</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Psi</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="wxCB">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>520</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wx</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="axCB">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>540</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ax</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="thetaCB">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>500</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Theta</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="thetaCB_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>500</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rudder</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="thetaCB_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>520</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Thrust</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="thetaCB_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>540</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearPlotButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>530</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="streamTab">
     <attribute name="title">
      <string>Stream</string>
     </attribute>
     <widget class="QLabel" name="streamImgLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string>Here be image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.77.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP-address:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QSlider" name="thrustSlider">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>22</width>
        <height>211</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-80</number>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
      <property name="value">
       <number>-80</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="rudderSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-5</number>
      </property>
      <property name="maximum">
       <number>55</number>
      </property>
      <property name="sliderPosition">
       <number>25</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="thrustLabel_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Thrust</string>
      </property>
     </widget>
     <widget class="QLabel" name="thrustLabel_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rudder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Const</string>
     </attribute>
     <widget class="QSpinBox" name="thrustSpinBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="rudderSpinBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>35</number>
      </property>
     </widget>
     <widget class="QLabel" name="thrustLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Thrust</string>
      </property>
     </widget>
     <widget class="QLabel" name="rudderLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rudder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadConstButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveConstButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>PID </string>
     </attribute>
     <widget class="QGroupBox" name="phiGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Phi</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="psiGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Psi</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_16">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_17">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_18">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="thetaGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Theta</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_22">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_23">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_24">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="wxGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Wx</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_25">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_26">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_27">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="wzGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Wz</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_28">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_29">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_30">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="wyGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Wy</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_31">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_32">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_33">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="savePidButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadConstButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="kpLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kp</string>
      </property>
     </widget>
     <widget class="QLabel" name="kiLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ki</string>
      </property>
     </widget>
     <widget class="QLabel" name="kdLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color: white;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color:white;</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cushionSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="cushionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cushion</string>
    </property>
   </widget>
   <widget class="QLabel" name="constTimeLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="constTimeSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <double>30.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
